packages/next/src/server/future/route-modules/app-page/module.compiled.d.ts

```diff
@@ -0,0 +1 @@
+export * from './module'
```

Comment:
This exports the type information from the module, as each of the runtimes are actually bundled route modules.